sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample03_BasicEventProcessing.cs

```diff
@@ -141,6 +166,24 @@ Task processEventHandler(ProcessEventArgs eventArgs)
 
             Task processErrorHandler(ProcessErrorEventArgs eventArgs)
             {
+                // As with the process event handler, the event arguments contain a cancellation token used by the Event Processor client to signal
+                // that the operation should be canceled.  The handler should respect cancellation as it is able in order to ensure that the Event
+                // Processor client is able to perform its operations efficiently.
```

Comment:
There's a segmentation here between "do the right thing" and "does that thing really matter".     You're correct in that there's no impact to the processor today, because we're using fire-and-forget semantics.   However, I'm considering that an implementation detail as we do not offer that as a guarantee or part of the formal contract for the handler.

In theory, best practice would have you always considering a cancellation token if you are provided with one.  Whether you deem it appropriate to respect that an interrupt your operation or decide that it is better to ignore it, you should at least acknowledge that there is a decision to be made.

My goal for including it is to try to present it as something that handler authors should be aware of and consider.  Likely, the practical answer to "should we stop if cancelled" when you're reporting an error is probably "no" - and the processor itself doesn't care.... but I think it's important to raise awareness that it is there and set an example of thinking about it.